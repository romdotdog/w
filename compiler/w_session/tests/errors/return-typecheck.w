fn main(): u32 {
    return 2.1;
    1
}