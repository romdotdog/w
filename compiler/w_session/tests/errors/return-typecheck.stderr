tests/errors/return-typecheck.w:2:12: error: type mismatch
    return 2.1;
           ^^^
